### PR TITLE
Restructure code

### PR DESCRIPTION
**Pull request by TarasMaliuchenko**
Add docker deployment
Add watchdog api
Add tests for watchdog api

### CODE REVIEW

File: module_type.json.sample [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  "configuration_spec": [
+    {
+      "key": "custom_key",
+      "type": "api_key",
+      "label": "API Key",
+      "tooltip": "The Palo Alto Networks AutoFocus API Key",
+      "required": true
+    }
+  ],
+  "capabilities": [
```
Comment (AnastasiiaRozlyvan) [inline module_type.json.sample:22]: I think `capabilities` and  `supported-apis` can be reduced to those that are listed in [Use Case](https://github.com/threatgrid/integrations/blob/master/Palo%20Alto/Use_Case_Palo_Alto.md).

Comment (TarasMaliuchenko) [reply module_type.json.sample:22]: Fixed

**Review by AnastasiiaRozlyvan — APPROVED** [review]

File: build.sh [hunk 2]
```diff
@@ -0,0 +1,31 @@
+#! /usr/bin/env sh
+echo " .:|:.:|:. "
+echo " C I S C O "
+echo "  SecureX "
+echo
+echo " Development Dockerfile build script."
+echo
+
+module_name="Palo Alto AutoFocus Relay"
```
Comment (AlinaBlahun) [inline build.sh:9]: Should it be just `Palo Alto AutoFocus`?

File: code/container_settings.json [hunk 3]
```diff
@@ -0,0 +1 @@
+{"VERSION": "1.0.0","NAME": "Palo Alto AutoFocus Relay"}
```
Comment (AlinaBlahun) [inline code/container_settings.json:1]: just `Palo Alto AutoFocus`?

File: build.sh (hunk 2 above)
Comment (TarasMaliuchenko) [reply build.sh:9]: Copied from README.md file

Comment (TarasMaliuchenko) [reply build.sh]: Fixed

File: code/container_settings.json (hunk 3 above)
Comment (TarasMaliuchenko) [reply code/container_settings.json]: Fixed

**Review by AlinaBlahun — APPROVED** [review]

**Pull request merged**